scene/resources/mesh_data_tool.h

```diff
@@ -37,6 +37,13 @@ class MeshDataTool : public RefCounted {
 	GDCLASS(MeshDataTool, RefCounted);
 
 	int format = 0;
+	union CustomVertexData {
+		_FORCE_INLINE_ CustomVertexData() {}
+		_FORCE_INLINE_ CustomVertexData(const Color &p_fd) { fd = p_fd; }
+		Color fd;
+		uint8_t ud[8];
```

Comment:
ud should be unnecesary and imo  CustomVertexData should not exist, only use Color.